test/integration/targets/vyos_user/tests/cli/basic.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
http://docs.ansible.com/ansible/dev_guide/testing_integration.html#conventions

I think this should be

remove test user
add test user
assert changed & result.commands
Add test user again (idempotent check
asset not changed

Collection of users
this is fine